examples/rendezvous/src/bin/rzv-identify.rs

```diff
@@ -19,48 +19,47 @@
 // DEALINGS IN THE SOFTWARE.
 
 use futures::StreamExt;
-use libp2p_core::{identity, upgrade::Version, Multiaddr, PeerId, Transport};
-use libp2p_identify as identify;
-use libp2p_ping as ping;
-use libp2p_rendezvous as rendezvous;
-use libp2p_swarm::{keep_alive, NetworkBehaviour, Swarm, SwarmEvent};
+use libp2p::{
+    core::transport::upgrade::Version,
+    identify, identity, ping, rendezvous,
+    swarm::{keep_alive, NetworkBehaviour, Swarm, SwarmEvent},
+    tcp, Multiaddr, PeerId, Transport,
+};
 use std::time::Duration;
-use void::Void;
 
 #[tokio::main]
 async fn main() {
     env_logger::init();
 
+    let key_pair = identity::Keypair::generate_ed25519();
     let rendezvous_point_address = "/ip4/127.0.0.1/tcp/62649".parse::<Multiaddr>().unwrap();
     let rendezvous_point = "12D3KooWDpJ7As7BWAwRMfu1VU2WCqNjvq387JEYKDBj4kx6nXTN"
         .parse()
         .unwrap();
 
-    let identity = identity::Keypair::generate_ed25519();
-
     let mut swarm = Swarm::with_tokio_executor(
-        libp2p_tcp::tokio::Transport::default()
+        tcp::tokio::Transport::default()
             .upgrade(Version::V1)
-            .authenticate(libp2p_noise::NoiseAuthenticated::xx(&identity).unwrap())
-            .multiplex(libp2p_yamux::YamuxConfig::default())
+            .authenticate(libp2p::noise::NoiseAuthenticated::xx(&key_pair).unwrap())
+            .multiplex(libp2p::yamux::YamuxConfig::default())
```

Comment:
Same with this topic here. I didn't explicitly comment on all lines but let's do this consistently across all examples and use `yamux::` etc in the code and `use libp2p::yamux` at the top. (For all `libp2p` modules: noise, yamux, etc)